shared/engine/errors.js

```diff
@@ -22,18 +22,11 @@ export default class EngineError extends Error {
 
     const niceFunc = niceMap[nameFix(err.name)]
     super(niceFunc && niceFunc(err) || err.desc || err.code)
```

Comment:
I think we shouldn't be using these directly since they affect things like the default toString() so maybe we just reach into raw if we really need it. could go either way